CHANGELOG.md

```diff
@@ -206,6 +206,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Multi DataSource] UX enhancement on Data source management stack ([#2521](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2521))
 - [Multi DataSource] UX enhancement on Update stored password modal for Data source management stack ([#2532](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2532))
 - [Monaco editor] Add json worker support ([#3424](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3424))
+- Fixing string placement in opensearch_dashboards_overview ([#3670](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3670))
```

Comment:
minor style nits - we prefer the imperative "use" or "fix" over "using" or "fixing". I also updated to be more reflective of the new feature.
```suggestion
- [Overview] Use branding.applicationTitle as page header ([#3670](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3670))
```

Reply:
Thank you for your feedback and updates, I'll make sure to use the imperative form and keep the new feature in mind for future responses.